templates/CRM/Birthdays/Form/Settings.tpl

```diff
@@ -0,0 +1,196 @@
+{* HEADER *}
+
+<div class="crm-submit-buttons">
+    {include file="CRM/common/formButtons.tpl" location="top"}
+</div>
+<h2>
+    {ts}General{/ts}
+</h2>
+<div id="help">{ts}This open source extension
+        ( <a href="https://github.com/systopia/de.systopia.birthdays#readme">de.systopia.birthdays</a> )
+        allows you to build a report on upcoming birthdays and provides an API for sending birthday
+        greetings automatically via e-mail.{/ts}
+    <br>
+    {ts}Please note: With default settings no email will be sent after upgrading to this version.
+        Mails only will be sent if triggering an APIv3 or APIv4 birthday action manually or using scheduled jobs.{/ts}
+</div>
+<br>
```

Comment:
Let's avoid using `<br>` in block level context.

Reply:
Lacking of a better approach